package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack"
```

Comment:
- [ ] **Optional:** To have your app run using **npm scripts** run the terminal command below:
   - `npm install -D webpack-dev-serve`
- update your scripts in `package.json` with the code snippet below
    ```
    "build": "webpack",
    "start": "webpack serve --open"
    ```